registration/approval.py

```diff
@@ -51,33 +48,61 @@ def _send_mail(application, template_name, subject):
         from_email=settings.BETATEST_EMAIL_SENDER,
         recipient_list=(application.user.email,),
     )
+    # Using a separate call to hide internal email
+    send_mail(
+        subject=subject,
+        message=message,
+        from_email=settings.BETATEST_EMAIL_SENDER,
+        recipient_list=(settings.BETATEST_EMAIL_INTERNAL,),
+    )
```

Comment:
You could replace both these calls with a single email with bcc.

Reply:
👍